core/utils.js

```diff
@@ -305,14 +341,42 @@ Blockly.getSvgXY_ = function(element) {
  * @param {string} name Element's tag name.
  * @param {!Object} attrs Dictionary of attribute names and values.
  * @param {Element=} opt_parent Optional parent on which to append the element.
+ * @param {Workspace=} workspace Optional workspace for access to context (scale...).
```

Comment:
Optional arguments must be named opt_workspace.
Also, the type would be {Blockly.Workspace=}